### PR TITLE
Changed Field Packet Updates

### PR DESCRIPTION
**Pull request by Clockworkx**
### Field Packet Updates

Broadcast all packets to players on Field once

### Added DungeonWait Packet.
It creates the display in the dungeon which says what kind of dungeon it is and for how many players it was created

### Misc

Added DungeonHandler Enum Type for entering dungeon lobby. This packet can / will probably be used to initiate trigger scripts
Added Check for send manual packet function, to prevent crashes on the server.

### CODE REVIEW

File: MapleServer2/PacketHandlers/Game/DungeonHandler.cs [hunk 1]
```diff
@@ -44,6 +48,10 @@ public override void Handle(GameSession session, PacketReader packet)
                     break;
             }
         }
```
Comment (AngeloTadeucci) [inline MapleServer2/PacketHandlers/Game/DungeonHandler.cs:50]: add line before the function

File: MapleServer2/Servers/Game/FieldManager.cs [hunk 2]
```diff
@@ -156,6 +156,16 @@ public IEnumerable<Packet> GetUpdates()
             }
             return updates;
         }
+        private void SendUpdates()
```
Comment (AngeloTadeucci) [inline MapleServer2/Servers/Game/FieldManager.cs:159]: same thing here

File: MapleServer2/Servers/Game/GameSession.cs [hunk 3]
```diff
@@ -31,23 +31,6 @@ public class GameSession : Session
         public GameSession(ManagerFactory<FieldManager> fieldManagerFactory, ILogger<GameSession> logger) : base(logger)
```
Comment (AngeloTadeucci) [inline MapleServer2/Servers/Game/GameSession.cs:31]: Remove the declaration of `CancellationToken` too!

**Review by AngeloTadeucci — CHANGES_REQUESTED** [review]

File: MapleServer2/PacketHandlers/Game/DungeonHandler.cs (hunk 1 above)
Comment (Clockworkx) [reply MapleServer2/PacketHandlers/Game/DungeonHandler.cs:50]: done!

File: MapleServer2/Servers/Game/FieldManager.cs (hunk 2 above)
Comment (Clockworkx) [reply MapleServer2/Servers/Game/FieldManager.cs:159]: done!

File: MapleServer2/Servers/Game/GameSession.cs (hunk 3 above)
Comment (Clockworkx) [reply MapleServer2/Servers/Game/GameSession.cs:31]: done!

**Review by AlanMorel — APPROVED** [review]

**Review by AngeloTadeucci — APPROVED** [review]

**Pull request merged**